vagrant/nodes.statedump.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: cluster:clients
+  become: yes
+  become_method: sudo
+  vars_files:
+    - ansible/config.yml
+  roles:
+    - logs.collect
+
+- hosts: cluster:clients
+  become: yes
+  become_method: sudo
+  vars_files:
+    - ansible/config.yml
+  roles:
+    - config.collect
```

Comment:
Maybe just adding the _config.collect_ role to the previous playbook is enough.

Reply:
Correct.